aireview/src/aireview/main.py

```diff
@@ -1,23 +1,23 @@
 import asyncio
+import logging
 
 from aireview.application.controllers.review_controller import ReviewController
 from aireview.config import Config
 from aireview.domain.services.code_analyzer import CodeAnalyzer
 from aireview.infrastructure.dust_client import DustClient
 from aireview.infrastructure.github_client import GitHubClient
 
+logger = logging.getLogger(__name__)
 
-
-def main():
+def main(pull_request: int):
     config = Config()
-    # TODO: create dust agent for reviews
+
     dust_client = DustClient(config.dust_api_key, config.claude_id, config.workspace_id)
     github_client = GitHubClient(config.github_token, "tehioant", "AIReview")
-
     analyzer = CodeAnalyzer(dust_client)
```

Comment:
**style**: Repository owner and name ('tehioant', 'AIReview') should be moved to configuration instead of being hardcoded